cli/cli/commands/run/run.go

```diff
@@ -588,6 +658,26 @@ func getOrCreateEnclaveContext(
 	return enclaveContext, isNewEnclaveFlagWhenCreated, nil
 }
 
+func getPackageDependencyYaml(
+	ctx context.Context,
+	enclaveCtx *enclaves.EnclaveContext,
+	starlarkScriptOrPackageId string,
+	isRemote bool,
+	packageArgs string,
+) (*kurtosis_core_rpc_api_bindings.PlanYaml, error) {
+	var packageYaml *kurtosis_core_rpc_api_bindings.PlanYaml
+	var err error
+	if isRemote {
+		packageYaml, err = enclaveCtx.GetStarlarkPackagePlanYaml(ctx, starlarkScriptOrPackageId, packageArgs, true)
```

Comment:
the hardcoded constant was for indicating only grabbing the dependencies but this will get removed once I process the yaml on the CLI said instead of in the APIC